src/test/java/spec/yandexSpec/YandexMailSpecs.java

```diff
@@ -0,0 +1,70 @@
+package spec.yandexSpec;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pages.yandex.HomePage;
+import pages.yandex.LogInPage;
+import pages.yandex.MailPage;
+import pages.yandex.YandexMainPage;
+import projectVeriables.ProjectVariables;
+import spec.BaseSpec;
+
+public class YandexMailSpecs extends BaseSpec {
+    private static final Logger logger = LoggerFactory.getLogger(YandexMailSpecs.class);
+    private HomePage homePage;
+
+    @Test
+    public void verifyYandexHomePage() {
+        homePage = new HomePage();
+        driver = homePage.getDriver();
+
+        Assertions.assertTrue(homePage.verifyHomePage());
```

Comment:
Please always add second parameter for assertTrue/assertFalse methods. If you will not add, then in report you will see message: "Expected true, but found false" and this case will be hard to understand what actually failed.